content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/_cq_design_dialog/.content.xml

```diff
@@ -57,6 +58,28 @@
                                     </clientlibs>
                                 </items>
                             </well>
+                            <heading1
+                                jcr:primaryType="nt:unstructured"
+                                sling:resourceType="granite/ui/components/coral/foundation/heading"
+                                level="{Long}4"
+                                text="Client-Side Libraries for Head element"/>
```

Comment:
What about CSS clientlibs? Currently this is not loaded. This should be clarified at least in the description.